src/libs/KeyboardShortcut/index.ts

```diff
@@ -130,12 +132,12 @@ function subscribe(
     key: string,
     callback: () => void,
     descriptionKey: string,
-    modifiers: string[] = ['shift'],
+    modifiers: ShortcutModifiers = ['CTRL'],
     captureOnInputs = false,
     shouldBubble = false,
     priority = 0,
     shouldPreventDefault = true,
-    excludedNodes = [],
+    excludedNodes: string[] | readonly never[] = [],
```

Comment:
@kubabutkiewicz Could you please clarify why `readonly never[]` is necessary?

Reply:
@VickyStash It because of that
<img width="507" alt="image" src="https://github.com/Expensify/App/assets/25230417/ddc3cb6a-70c5-446b-aa8f-dd43ea695961">
we are setting default value which comes from `CONST` and its basically an empty array but kt comes from `CONST` thats why its need to be typed as `readonly never[]`